packages/infinite-loop-loader/README.md

```diff
@@ -0,0 +1,60 @@
+# infinite-loop-loader
+[![Build Status](https://travis-ci.org/nickbalestra/infinite-loop-loader.svg?branch=master)](https://travis-ci.org/nickbalestra/infinite-loop-loader)
```

Comment:
Good catch! This could be removed, as we have now the travis badge on root for the whole monorepo